util/find-doc-nits

```diff
@@ -387,7 +388,19 @@ my %preferred_words = (
     'useable'       => 'usable',
     'userspace'     => 'user space',
     'user name'     => 'username',
-    'zeroes'        => 'zeros'
+    'zeroes'        => 'zeros',
+    # https://ac.bslw.com/community/wiki/index.php/British_to_American_Spelling
+    'behavior'      => 'behaviour',
+    'enrollment'    => 'enrolment',
+    'favor'         => 'favour',
+    'finalize'      => 'finalize',
+    'finalization'  => 'finalisation',
```

Comment:
surely initialisation must be used somewhere?

Reply:
wow, good catch.  will add it and push a new commit.